spec/system/kudo_manage_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'Kudo management', type: :system do
+  let(:employee) { create(:employee) }
+  let!(:kudo) { create(:kudo, giver: employee) }
+
+  before do
+    login_as(employee, scope: :employee)
+    driven_by(:selenium_chrome_headless)
+  end
+
+  it 'enables me to edit kudos' do
+    visit root_path
+    click_button 'Edit'
+    fill_in 'Title', with: 'A new title'
+    fill_in 'Content', with: 'A new content'
+    click_button 'Update Kudo'
+    expect(page).to have_text('Kudo was edited successfully')
```

Comment:
it would be nice to check if edited values were saved
I would add one more line with `expect(page).to have_text('A new title')`